ocaml/xenopsd/doc/walk-throughs/VM.migrate.md

```diff
@@ -0,0 +1,176 @@
+Walk-through: Migrating a VM
+============================
+
+# From client to Xenopsd
+
+A Xenopsd client wishes to migrate a VM from one host to another within
```

Comment:
Do you mean a xapi client? They way to control VMs from outside the host is through xapi, not xenopsd.

Reply:
yes absolutely, I mean xapi client.